### DISCUSSION TITLE
`0.7.5` Duplicate Transfer-Data-Plane-Signaling Dependencies in `edc-controlplane-base` Module?

### DISCUSSION THREAD

**Discussion by efiege** [eclipse-tractusx/tractusx-edc#1576] [Q&A] [answered]
During our review of the dependency tree for the `edc-controlplane-postgresql-azure-vault` module, we observed that both the following dependencies are being included:

- org.eclipse.edc:transfer-data-plane-signaling:0.7.2
- project :edc-extensions:transfer-dataplane-signaling

It appears that the version included in this repository is a patched version of `org.eclipse.edc:transfer-data-plane-signaling:0.7.2`, addressing this issue: https://github.com/eclipse-tractusx/tractusx-edc/pull/1534.

However, having both dependencies included might result in the registration of two `DataPlaneSignalingFlowController` instances within the `DataFlowManager`, one with the fix and one without. This could potentially cause the EDC to load the unpatched version, depending on how it resolves these extensions.

The unpatched core EDC version is referenced here:

https://github.com/eclipse-tractusx/tractusx-edc/blob/75d1526115bf3922c204aea4ff8434a1b37eb514/gradle/libs.versions.toml#L233

https://github.com/eclipse-tractusx/tractusx-edc/blob/75d1526115bf3922c204aea4ff8434a1b37eb514/edc-controlplane/edc-controlplane-base/build.gradle.kts#L70

**Comment by wolf4ood** [comment, marked as answer]
Hi @efiege 

thanks for the discussion, this seems an issue, i will create an issue from this

Thanks
